src/main/java/org/sciborgs1155/robot/Drive/SimDrive.java

```diff
@@ -0,0 +1,66 @@
+package org.sciborgs1155.robot.Drive;
+
+import edu.wpi.first.math.VecBuilder;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.system.plant.DCMotor;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.wpilibj.simulation.DifferentialDrivetrainSim;
+import edu.wpi.first.wpilibj.smartdashboard.Field2d;
+import monologue.Monologue.LogBoth;
+import org.sciborgs1155.robot.Constants;
+
+public class SimDrive implements DriveIO {
+
+  Pose2d pose = new Pose2d();
+
+  DifferentialDrivetrainSim sim =
+      new DifferentialDrivetrainSim(
+          DCMotor.getNEO(2),
+          9,
+          8.5,
```

Comment:
put these random constants in the constants file (actually you should probably go through your code and just scan for things that should be in DriveConstants)

Reply:
ok